quinn-proto/src/congestion.rs

```diff
@@ -56,9 +56,13 @@ pub trait Controller: Send + Sync {
         now: Instant,
         sent: Instant,
         is_persistent_congestion: bool,
+        is_ecn: bool,
         lost_bytes: u64,
     );
 
+    /// Packets were incorrectly deemed lost.
```

Comment:
Let's expand on exactly when this is called, and discuss some corner cases. In particular: what happens if two separate congestion events occur, and then the packet loss from the *first* event is found to be spurious?

Perhaps we don't want to commit to a specific behavior here, but we should be clear about what can be relied on.